src/Organizer/Organizer.php

```diff
@@ -340,6 +340,42 @@ public function updateImage(
         );
     }
 
+    public function importImages(Images $images): void
+    {
+        $currentImages = $this->images->toArray();
+        $importImages = $images->toArray();
+
+        $compareImages = static function (Image $a, Image $b) {
```

Comment:
Nit: maybe use the following shorthand
```
$compareImages = static fn (Image $a, Image $b) => strcmp($a->getId()->toString(), $b->getId()->toString());
```

Reply:
Yeah good suggestion. I just copied this from `Offer::importImages()` :)